api/src/main/java/io/hyperfoil/api/config/PhaseReferenceDelay.java

```diff
@@ -0,0 +1,10 @@
+package io.hyperfoil.api.config;
+
+public class PhaseReferenceDelay extends PhaseReference {
```

Comment:
I don't feel very satisfied about this `PhaseReferenceDelay` class name, but I had to differentiate it with respect to `StartWithDelay` object (the one actually used in the Phase). I was thinking about adding optional `delay` field in the `PhaseReference` in order to avoid some duplication.. but I did not want to mix things up, wdyt?